app/representers/api/api_representer.rb

```diff
@@ -78,4 +78,12 @@ class Api::ApiRepresenter < Api::BaseRepresenter
     ]
   end
 
+  link :authorization do
+    {
+      title: "Get information about the active authorization for this request",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.